src/api/api.c

```diff
@@ -1218,19 +1218,11 @@ int8_t z_get(const z_loaned_session_t *zs, const z_loaned_keyexpr_t *keyexpr, co
     return ret;
 }
 
-_Bool z_reply_is_ok(const z_loaned_reply_t *reply) {
-    _ZP_UNUSED(reply);
-    // For the moment always return TRUE.
-    // FIXME: The support for reply errors will come in the next release.
-    return true;
-}
+_Bool z_reply_is_ok(const z_loaned_reply_t *reply) { return reply->_tag != _Z_REPLY_TAG_ERROR; }
```

Comment:
With this modification, this PR probably closes #345 as well.

Reply:
Seems, yes